codechecker_lib/analyzers/result_handler_base.py

```diff
@@ -183,17 +183,24 @@ def get_analyzer_result_file(self):
                 '_' + analyzed_file_name + '_' + uid + '.plist'
 
             out_file = os.path.join(self.__workspace, out_file_name)
-            self.__res_file = out_file
+            self.__result_file = out_file
+
+        return self.__result_file
 
-        return self.__res_file
+    @analyzer_result_file.setter
+    def result_file(self, file_path):
+        """
+        The source file which is analyzed.
```

Comment:
Is this a source file or a result file, the naming and the comment is misleading.